c16 translation (#113)
IllusionMods/KoikatsuTranslation: translation/en/RedirectedResources/assets/abdata/h/list/20_00/personality_voice_c16_03_02/translation.txt

//
// Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
//

> いつまで自分のモノを晒しているつもり？　私の中に挿れて、隠さなきゃダメよ……=How much longer do you plan on exposing your dick for? You gotta hide it, inside me...

Reviewer: ```suggestion
いつまで自分のモノを晒しているつもり？　私の中に挿れて、隠さなきゃダメよ……=How long are you going to stand there with your dick out? You gotta hide it, inside me...
```
